docs/modules/secret-operator/pages/security.adoc

```diff
@@ -0,0 +1,16 @@
+= Security
+
+== Container privileges
+
+By default, the Secret Operator runs as a set of privileged root containers. This is done for three reasons:
+
+1. We need to run as root to have permission to create the Unix Domain Socket hosting the Container Storage interface (CSI)
+   driver.
+2. We need to run as root to have permission to write secret material into the pods' volume paths, as directed
+   by the CSI.
+3. We need to run as a privileged container in order to be able to mount the volume folders as RAM disks, in order to avoid
+   spilling secret material onto disk.
+
+Running as root is currently a hard requirement. Secret Operator _can_ run in an unprivileged container (by passing
+`--set securityContext.privileged=false` to `helm upgrade`), but doing so is strongly discouraged, and users
```

Comment:
If anyone were not using Helm - and I'm sorry if this is a stupid question - how would it work then?
Is this purely a function of the securityContext and the operator detects whether it runs privileged or not?

Reply:
It's controlled by either the CLI argument `--privileged`, or the environment variable `PRIVILEGED`.